projects/FIFE.yaml

```diff
@@ -0,0 +1,11 @@
+Department: N/A
+Description: Project entry corresponding to FIFE experiments within the Fermilab VO.
+FieldOfScience: High Energy Physics
+ID: ''
```

Comment:
@shreyb before we merge this into master, could you approve this suggestion?

```suggestion
ID: '564'
```

Reply:
Yep - done.